apps/core/scenes/search/SearchContext.js

```diff
@@ -6,6 +6,7 @@ import * as DateFNS from 'date-fns';
 
 import { MODAL_TYPE } from './SearchConstants';
 
+// TODO move types to separate file
```

Comment:
Why?

Reply:
Thanks for asking, I added this note, and wait if someone has an opinion about that. My idea behind this is, that exists a lot of type definitions already in this file, so I was thinking split it to two files one for code and one for types. It should be more readable but on another side in the current state is easy to import these types together with context.

Reply:
The types defined here are describing the state of the Context, nothing more, nothing less, and IMHO it belongs in the same file as the context itself.

Reply:
OK let's keep these types here.